docs/userguide.md

```diff
@@ -60,7 +60,10 @@ Once you have minikube installed run:
 ```bash
 minikube start -p cluster1 --kubernetes-version v1.11.0
 minikube start -p cluster2 --kubernetes-version v1.11.0
-kubectl config use-context cluster1
+```
+**NOTE:** Please make sure to run below command otherwise the rest of the guide will not work.
```

Comment:
I would modify the verbiage a little by stating something like: *Please make sure to set the correct context using the command below as this guide depends on it.*

Reply:
Sure thing! now it makes more sense.